src/core/osmLayer.js

```diff
@@ -43,6 +43,7 @@ geo.osmLayer = function (arg) {
     m_numberOfCachedTiles = 0,
     m_tileCacheSize = 100,
     m_baseUrl = 'http://tile.openstreetmap.org/',
+    m_subdomains = 'abc',
```

Comment:
This really isn't general enough.  See [here](http://wiki.openstreetmap.org/wiki/Slippy_map_tilenames#Tile_servers)